.travis/runcollection.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
I think you can do ` #!/bin/bash -e` instead of using `set -e` later.

Reply:
sure - if you want, please propose a change to the source script at https://github.com/linux-system-roles/template/blob/master/.travis/runcollection.sh then we can again copy it to every other system role repo